src/cljs/rems/application.cljs

```diff
@@ -584,7 +592,7 @@
                               :contents [format-validation-errors application errors]}])])]
     [:div
      [:div {:class "float-right"} [pdf-button (:application/id application)]]
-     [document-title (text :t.applications/application)]
+     [document-title [:span (text :t.applications/application) " " (:application/external-id application)]]
```

Comment:
If UI is configured to show id instead of external-id in the application list (determined by :application-id-column in config), it would probably be good to do it here, as well.

Reply:
Indeed!